frontend/src/pages/zaaktype/components/zaaktypetabs/ZaaktypeAttributeGridTab.tsx

```diff
@@ -82,11 +141,30 @@ export const ZaaktypeAttributeGridTab = ({
       const fieldName = _fieldName as keyof TargetType;
       const originalValue = object[fieldName];
 
-      if (
-        originalValue &&
-        !isPrimitive(originalValue) &&
-        !Array.isArray(originalValue)
-      ) {
+      if (!originalValue || isPrimitive(originalValue)) continue;
+
+      // When deelzaaktypen we want to render badges for selected options
+      if (fieldName === "deelzaaktypen") {
+        overrides[fieldName] = isEditing
+          ? (originalValue as string[])
+          : selectedDeelzaaktypenOptions.length > 0
+            ? selectedDeelzaaktypenOptions.map((option) => (
+                <Badge
+                  key={option.value}
+                  href={
```

Comment:
We can pass the direct link for robustness, but won't this bypass react router? We should probably wrap this into NavLink or some sort of integration with React Router.